0007/README.md

```diff
@@ -0,0 +1,113 @@
+---
+id: 0007
+title: Sandbox and the user persona
+status: discussion
+authors: Gianluca Arbezzano <gianarb92@gmail.com>
+---
+
+## Summary
+
+The setup workflows and tutorial we have today in my opinion are for
+contributors. For developers who want to code for Tinkerbell.
+
+Because the process checkouts the `tinkerbell/tink` repository, and it runs the latest version of
+dependencies like: osie, tink-server, tink-worker, boots and so on. This is a
+big problem today because all those components are under development, and we
+can't guarantee that what it is in master works. Even less in integration with
+the other components. The entropy is too high.
+
+## Goals and non-Goals
+
+Goals:
+
+1. Figure out a way to serve a pinned "version" of Tinkerbell that does not
+   use the current master version but something that is tested.
+2. Avoid compiling or building code or docker images in favor of using already
+   available solutions.
+
+Non-Goals:
+
+1. Tag a release for project, we are not ready for that and there is proposal
+   0002 under discussion about that.
+2. Getting to something that will work forever, we need something that works
+   NOW.
+3. Make the contributor experience better
+
+## Content
+
+I made a prototype
+[gianarb/sandbox](https://github.com/gianab/playwithtink).
```

Comment:
change this before merge with the actual repo tinkerbell/sandbox

Reply:
Also - this username has a typo